File: lesson_13/hometask/Homework_12_6/product/Cocktail.java
package lesson_13.hometask.Homework_12_6.product;

// Дабавляем новый класс-наследник не изменяя реализацию класса-предшественника

public class Cocktail extends Drink{
private double abv;

    public Cocktail(String name, int volume, double abv) {
        super(name, volume);
        this.abv = abv;
    }

    public double getABV() {
        return abv;
    }

    public void setABV(double abv) {
        this.abv = abv;
    }   
   
    @Override
    public String toString() {
        return String.format("%s, volume %d, Alcohol %s", name, volume, abv);

    }

}

File: lesson_13/hometask/HW_py/Cocktail.py
from Drink import Drink

class Cocktail(Drink):

    def __init__(self, name, volume, abv):
        self._name = name
        self._volume = volume
        self._ABV = abv

    def getTemp(self):
        return self._ABV
    
    def setTemperature(self, abv):
        self._ABV = abv
    
    def __str__(self):
        return f"{self._name} volume {self._volume} alcohol {self._ABV}" 